packages/docs/components/link.md

```diff
@@ -25,148 +22,121 @@ links:
+Links may be used within content or as standalone UI. When using links within content - like a paragraph or list item - do not pair them with Icons.
+
+Avoid using Links for actions, preferring Buttons instead.
 
 </Description>
 
-#### Max length
+## Content guidelines
 
 <Description>
 
-Try to limit a link to at most 3 words.
+Choose Link copy that describes the destination (e.g. "Settings"), rather than generic text (e.g. "Click here" or a URL). Aim to keep this copy concise, three words at most.
+
+Keep in mind that all users may not have the same visual context due to their device or other constraints.
 
 </Description>
 
 <Visual variant="positive">
   <template>
-    <a href="#">View menu</a>
+    <a href="#">View solar system</a>
   </template>
 </Visual>
 
 <Visual variant="negative">
   <template>
-    <a href="#">View all available donuts</a>
+    <a href="#">View</a>
+  </template>
+</Visual>
+
+<Visual variant="negative">
+  <template>
+    <a href="#">View all local stellar orbiters</a>
   </template>
 </Visual>
 
-### Context
+### Icons
 
 <Description>
 
-Choose link text that describes the destination (e.g. "Settings"), rather than generic text (e.g. "Click here" or a URL)
-. This is particularly important for assistive technologies like screen readers.
+Icons may be included in standalone links. They are not supported within paragraph content or longer copy.
 
-If you must have brief link text, ensure that additional context is provided by using the `.u-visually-hidden` class.
+Icon layout is automatic, based on language direction.
 
 </Description>
 
-<Visual variant="positive">
-  <template>
-    <a href="#">View all reports</a>
-  </template>
-</Visual>
+<Visual>
+
+<a href="#icons" class="ods-link--has-icon"><svg aria-hidden viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" class="ods-icon"><path fill-rule="evenodd" clip-rule="evenodd" d="M7 13C10.3137 13 13 10.3137 13 7C13 3.68629 10.3137 1 7 1C3.68629 1 1 3.68629 1 7C1 10.3137 3.68629 13 7 13ZM8 4C8 4.55228 7.55228 5 7 5C6.44772 5 6 4.55228 6 4C6 3.44772 6.44772 3 7 3C7.55228 3 8 3.44772 8 4ZM8 6V11H6V6H8Z" fill="currentColor"/></svg>Learn about propulsion</a>
 
-<Visual variant="negative">
-  <template>
-    <a href="#">View</a>
-  </template>
 </Visual>
 
-### Semantics
+### Mailto
 
 <Description>
 
-Avoid using a link <code>&lt;a&gt;</code> for actions; use a button <code>&lt;button&gt;</code> instead
+If you need a direct email link, display the whole address (e.g. lauren.ipsum@okta.com).
 
-</Description>
+Avoid colloquial text that hides the associated email (e.g. "Contact Us").
 
-<Visual variant="positive">
-  <template>
-    <a href="#">View all reports</a>
-  </template>
-</Visual>
+</Description>
 
-<Visual variant="negative">
+<Visual>
   <template>
-    <a href="#">View</a>
+    <a href="mailto:odysseus@okta.com">odysseus@okta.com</a>
```

Comment:
just curious: is this a real email address?

Reply:
it's not - but maybe it should be?